code/modules/jobs/job_types/legion.dm

```diff
@@ -137,12 +180,10 @@
 	exp_requirements = 1500
 
 	loadout_options = list(
-	/datum/outfit/loadout/palacent, //10mm SMG, large magazines and thermic lance
-	/datum/outfit/loadout/rangerhunter, //hunting revolver and ripper
-	/datum/outfit/loadout/centurion //marksman and powerfist
+	/datum/outfit/loadout/palacent,			// 10mm SMG, Thermic lance
+	/datum/outfit/loadout/rangerhunter,		// Hunting revolver, Ripper
+	/datum/outfit/loadout/centurion			// M1919 Machinegun, Powerfist
```

Comment:
```suggestion
	/datum/outfit/loadout/palacent, // 10mm SMG, Thermic lance
	/datum/outfit/loadout/rangerhunter, // Hunting revolver, Ripper
	/datum/outfit/loadout/centurion, // M1919 Machinegun, Powerfist
```
* Make use of trailing commas.
* Avoid excessive horizontal drift. Remember tabs are not the same in every editor.